code/modules/mob/living/simple_animal/abnormality/aleph/nobody_is.dm

```diff
@@ -311,17 +401,30 @@
 		new /obj/effect/temp_visual/cult/sparks(T)
 	var/whip_delay = (get_dist(src, target) <= 2) ? (0.75 SECONDS) : (0.5 SECONDS)
 	SLEEP_CHECK_DEATH(whip_delay)
-	for(var/i = 1 to whip_count)
-		var/obj/projectile/P = new /obj/projectile/beam/nobody(start_loc)
-		P.starting = start_loc
-		P.firer = src
-		P.fired_from = src
-		P.yo = target_loc.y - start_loc.y
-		P.xo = target_loc.x - start_loc.x
-		P.original = target
-		P.preparePixelProjectile(target_loc, src, spread = rand(0, 10))
-		P.damage = whip_damage
-		P.fire()
+	if(oberon_mode)
```

Comment:
Instead of copying the whole code you can just do an if statement for the projectile, I'm fairly certain